app/src/org/commcare/fragments/personalId/PersonalIdPhoneVerificationFragment.java

```diff
@@ -281,6 +306,9 @@ public void onDestroyView() {
     public void onSaveInstanceState(@NonNull Bundle outState) {
         super.onSaveInstanceState(outState);
         outState.putString(KEY_PHONE, primaryPhone);
+        outState.putString(KEY_LAST_OTP_METHOD, lastOtpMethod);
+        outState.putBoolean(KEY_VERIFY_BUTTON_ENABLED, binding.connectPhoneVerifyButton.isEnabled());
+        outState.putString(KEY_OTP_REQUEST_TIME_STRING, otpRequestTime.toString());
```

Comment:
Confirming `otpRequestTime` can't be null here ? Mostly wondering if the activity can get destroyed before we init it ?

Reply:
Good question - it being null is very unlikely but technically, `otpRequestTime` will be null until we init it via `onCreateView()` (which is the worst case scenario I think).

Since it _is_ possible, I'll add a null check to `restoreState()` when we parse the time string to avoid an unnecessary crash